sources/packages/backend/libs/sims-db/src/entities/sfas-part-time-application.model.ts

```diff
@@ -56,4 +56,45 @@ export class SFASPartTimeApplications extends BaseSFASApplicationModel {
     nullable: false,
   })
   extractedAt: Date;
+  /**
+   * CSPT Award Amount (sail_extract_data.sail_cspt_amt).
+   */
+  @Column({
+    name: "cspt_award",
+    nullable: true,
+  })
+  csptAward: number;
```

Comment:
Please mark the field as optional if it is nullable true. `e.g. csptAward?: number;`